cypress/integration/prescription-spec.js

```diff
@@ -0,0 +1,70 @@
+describe('Prescription', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://unstuck-rubberduck-api.herokuapp.com/api/v1/rr/rest', {
+      body: {
+        selection: {
+          id: 1,
+          type: ['rest'],
+          name: "some Oblique R&R",
+          content: "Do nothing for as long as possible.",
+          resource: 'https://en.wikipedia.org/wiki/Oblique_Strategies',
+          contentMedium: 'text',
```

Comment:
conflicted about testing ~10 possible fetches given that they all require a different form input sequence